### ISSUE TITLE
Scaling issue - Toolbar and Pen width panel

### ISSUE THREAD

**Issue by BuzaL** [geovens/gInk#182]
I using a portrait + landscape monitor.
On portrait monitor, the default toolbar size scaled to the height, instead of "smaller" side.
I handled it with Toolbar_Height = 0.02 settings, but the "Pen width" icon, and the width bar does not fit into the bar.
![image](https://user-images.githubusercontent.com/29695425/203558137-eac46ea1-f00f-4f9b-82f0-5d850395e676.png)

**Comment by geovens** [comment]
In case that you are still concerned with this issue: Have you tried the following tip and does it make any difference?

> There is a known issue for multiple displays of unmatched DPI settings (100%, 125%, 150%, etc.). If you use gInk on a computer with multiple displays of unmatched DPI settings, or you encounter problems such as incorrect snapshot area, being unable to drag toolbar to locations etc., please do the following as a workaround (in Windows 10 version 1903 as an example): right-click gInk.exe, Properties, Compatibility, Change high DPI settings, Enable override high DPI scaling behavior scaling performed by: Application. (do this only for gInk version v1.1.0 and after)

**Comment by BuzaL** [comment]
> In case that you are still concerned with this issue: Have you tried the following tip and does it make any difference?
> 
> > There is a known issue for multiple displays of unmatched DPI settings (100%, 125%, 150%, etc.). If you use gInk on a computer with multiple displays of unmatched DPI settings, or you encounter problems such as incorrect snapshot area, being unable to drag toolbar to locations etc., please do the following as a workaround (in Windows 10 version 1903 as an example): right-click gInk.exe, Properties, Compatibility, Change high DPI settings, Enable override high DPI scaling behavior scaling performed by: Application. (do this only for gInk version v1.1.0 and after)

Yes, i've tried, but doesn't help at all. The reason is not the DPI but the aspect ratio, cause the main monitor is 10:16 (rotated), but the secondary is 16:10 (these monitors are same: Samsung SA450)

**Comment by BuzaL** [comment]
normal cursor
![image](https://user-images.githubusercontent.com/29695425/218977734-c021e40f-115f-4235-bf9d-1acb7f1bbd2a.png)
gInk cursor
![image](https://user-images.githubusercontent.com/29695425/218977879-da17a641-cee1-4b91-948f-29fc9e3419b7.png)
